app/models/manageiq/providers/redhat/infra_manager/vm.rb

```diff
@@ -6,9 +6,9 @@ class ManageIQ::Providers::Redhat::InfraManager::Vm < ManageIQ::Providers::Ovirt
 
   supports :migrate do
     if blank? || orphaned? || archived?
-      unsupported_reason_add(:migrate, "Migrate operation in not supported.")
+      "Migrate operation in not supported."
     elsif !ext_management_system.supports?(:migrate)
-      unsupported_reason_add(:migrate, 'RHV API version does not support migrate')
+      'RHV API version does not support migrate'
```

Comment:
Could this be?
```
else
  ext_management_system.unsupported_reason(:migrate)
```

to prevent copying the unsupported reason string in two places?